src/features/agenda/queries/lectureHooks.ts

```diff
@@ -1,4 +1,5 @@
 import { useMemo } from 'react';
+import hex_md5 from 'react-native-uuid/dist/md5';
```

Comment:
this looks quite like a dangerous import.... is there no other better way to do it?